src/Routes.js

```diff
@@ -7,8 +7,9 @@ import HomePage from "./pages/HomeView";
 import CoursePage from "./pages/CoursePageViewWithVideo";
 import WishlistPage from "./pages/wishListView";
 import ProfilePage from "./pages/ProfilePage";
+import Login from './pages/Login/Login';
 
-const renderRoutes = () => (
+let renderRoutes = () => (
```

Comment:
Why this change?

Reply:
I am Rendering all routes only if a person is logged in , so I have to change renderRoutes if a person is nnot logged in that is why i make it a let instead of const